internal/app/handler/handler.go

```diff
@@ -73,8 +73,7 @@ func NewGinEngine(
 			Str(logcontext.KeyHostname, md.Hostname).
 			Logger()
 
-		//newCtx := sublogger.WithContext(c.Request.Context())
-		newCtx := logging.ToContext(c.Request.Context(), sublogger)
+		newCtx := logging.ToContext(c, sublogger)
```

Comment:
`nrgin` middleware only stores the `txn` (New Relic transaction) in the Gin Context, but not in the underlying HTTP request context.

That's why we have to use `c` (the Gin Context) here instead of `c.Request.Context()` (the HTTP request context).

The HTTP request's context is updated to include both the logger and the New Relic transaction.